tests/lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -2489,27 +2490,27 @@ ruleTester.run(
                     code: "obj.import(source)",
                     options: [{ version: "12.0.0" }],
                 },
-                {
+                ...["12.17.0", "13.2.0"].map(v => ({
                     code: "import(source)",
-                    options: [
-                        { version: "13.1.0", ignores: ["dynamicImport"] },
-                    ],
-                },
+                    options: [{ version: v }],
+                })),
             ],
             invalid: [
-                {
+                ...["12.16.0", "13.0.0", "13.1.0"].map(v => ({
                     code: "import(source)",
-                    options: [{ version: "13.3.0" }],
+                    options: [{ version: v }],
                     errors: [
                         {
                             messageId: "no-dynamic-import",
                             data: {
-                                supported: null,
-                                version: "13.3.0",
+                                supported: new Range(
+                                    ">=12.17 <13 || >=13.2"
+                                ).toString(),
```

Comment:
I figured it was better to configure this with whatever `Range`'s behavior is so that we didn't end up being brittle around `Range` when we really just wanted to test our own code.